<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerNCIMultiVolumeRayCastVolumeRenderingPropertiesWidget</class>
 <widget class="QWidget" name="qSlicerNCIMultiVolumeRayCastVolumeRenderingPropertiesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NCI Multi Volume RayCast</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::FieldsStayAtSizeHint</enum>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QLabel" name="RenderingTechniqueLabel">
     <property name="text">
      <string>Technique (bg):</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="RenderingTechniqueComboBox">
     <property name="toolTip">
      <string>Select GPU ray casting technique for bg volume.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="RenderingTechniqueFgLabel">
     <property name="text">
      <string>Technique (fg):</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="RenderingTechniqueFgComboBox">
     <property name="toolTip">
      <string>Select GPU ray casting technique for fg volume.</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="FusionLabel">
     <property name="text">
      <string>Fusion:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="FusionComboBox">
     <property name="toolTip">
      <string>Select color fusion method in multi-volume rendering</string>
     </property>
     <item>
      <property name="text">
       <string>Alpha Blending (OR)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Alpha Blending (AND)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Alpha Blending (NOT)</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="BgFgRatioLabel">
     <property name="text">
      <string>Bg|Fg ratio:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="ctkSliderWidget" name="BgFgRatioSliderWidget">
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="pageStep">
      <double>0.200000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>0.500000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="DepthPeelingLabel">
     <property name="text">
      <string>Vol. Depth Peeling:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="ctkSliderWidget" name="DepthPeelingSliderWidget">
     <property name="toolTip">
      <string>Set threshold for Volumetric Depth Peeling. Volume rendering starts after we have met scalar values higher than the threshold. Use with transfer functions together.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
